Scripts/Helpers/EditorWindowCapture.cs

```diff
@@ -99,6 +117,60 @@ private void Update()
 				m_GrabPixels.Invoke(m_GuiView, new object[] { texture, rect });
 			}
 		}
+
+		public void SendEvent(Transform rayOrigin, Transform workspace, EventType type, Vector2 offset = default(Vector2))
+		{
+			if (m_Window == null)
+				return;
+
+			var ray = new Ray(rayOrigin.position, rayOrigin.forward);
+			var plane = new Plane(workspace.up, workspace.position);
+			float distance;
+			plane.Raycast(ray, out distance);
+			var localPosition = workspace.InverseTransformPoint(ray.GetPoint(distance));
+			var worldCorners = new Vector3[4];
+			m_RectTransform.GetWorldCorners(worldCorners);
+			var contentSize = new Vector2((worldCorners[1] - worldCorners[2]).magnitude,
+				(worldCorners[0] - worldCorners[1]).magnitude);
+			localPosition.x /= contentSize.x;
+			localPosition.z /= -contentSize.y;
+			var rectPosition = new Vector2(localPosition.x + 0.5f, localPosition.z + 0.5f);
+
+			var rect = m_Window.position;
+			var clickPosition = Vector2.Scale(rectPosition, rect.size) + offset;
+
+			const int windowHeaderSize = 25;
```

Comment:
Let's figure out what is causing the additional offset between setting a mouse position here and sending the event.